### DISCUSSION TITLE
Are there any unsupported features

### DISCUSSION THREAD

**Discussion by kirtangajjar** [harness-software/wp-graphql-gravity-forms#247] [Q&A] [answered]
Hey there 👋,

First of all, kudos to the team for the awesome work on this plugin. It is very helpful and seems to cover almost all use cases of GF in a headless environment! 

My question was that I was just having a look at the plugin and although it looked complete, it wasn't immediately clear to me what fields weren't supported.

So can we do everything with this plugin that we can do with REST API? or are there any caveats that we need to handle manually? I'm asking this especially since I read in [an article](https://developers.wpengine.com/blog/work-with-gravity-forms-in-headless-wordpress), that there are some fields that the plugin doesn't support(or need a workaround?).

There's a good chance that the situation described in the blog has changed so I thought it's best to ask here :)

**Comment by justlevine** [comment, marked as answer]
Hey @kirtangajjar 

While we're getting closer to feature parity with GF, we're not there yet. We're still missing pricing fields and the beta repeater field from core, and several official addons. 

Here's the list of [supported form fields](https://github.com/harness-software/wp-graphql-gravity-forms/blob/develop/docs/form-field-support.md#supported--experimental--and-unsupported-fields) and the [tracker for remaining fields](https://github.com/harness-software/wp-graphql-gravity-forms/issues/195).

PS: v0.10.0 saw the schema undergo a major rewrite. The principles in article you linked to still apply, but the actual graphql they used is out of date.

**Reply by kirtangajjar** [reply]
Thanks a lot for this answer @justlevine. It definitely cleared things up for me :)
